### ISSUE TITLE
Foundation slows my machine way down

### ISSUE THREAD

**Issue by devemmastone** [foundation/foundation-sites#12101]
Hello all,
I am new to Foundation and want to work with the Scss version of the framework. I have a folder with the files for Foundation in it and then have my sites call in Foundation, all good till I use `@include foundation-everything;` It just slows my machine down. I am running Hugo and that even times out on me. I tried to also include each of the components but as soon as I came to the button components it slows down my machine again. Also it gives me a warning saying the colours are not good and then crashes Hugo.

What am I doing wrong? I could just use the CDN for the CSS but want an easy way to override the styles.

Thanks in advance.

**Comment by DanielRuf** [comment]
tl;dr: please disable any antivirus solution, especially the realtime scanner features (Defender / Windows and others).

**Comment by DanielRuf** [comment]
Oh and please check your node-sass / sass version, there are known issues with outdated versions.

**Comment by devemmastone** [comment]
I am on Linux and don't have any antivirus software running, also have Scss up-to-date.
When I add `@include foundation-button-group;` it just kills my development machine and times it out. I want to load all of Foundation and then use my own Scss settings to override the defaults.

**Comment by DanielRuf** [comment]
Please check the comments in https://github.com/foundation/foundation-sites/issues/12070

This is a problem with the sass compiler version.

Closing as duplicate.
